hypervisor/include/hw/pci.h

```diff
@@ -257,11 +257,6 @@ static inline bool is_host_bridge(const struct pci_pdev *pdev)
 	return (pdev->base_class == PCIC_BRIDGE) && (pdev->sub_class == PCIS_BRIDGE_HOST);
 }
 
-static inline bool is_bridge(const struct pci_pdev *pdev)
```

Comment:
This API is introduced by your previous patch. Are u sure you want to remove?

The rest APIs should be removed.

Reply:
I'm not sure. I prefer we keep is_pci_typeXXX(struct pci_pdev *pdev) these APIs than is_pci_cfg_multifunction or is_pci_cfg_bridge.

Reply:
You decide.
to me, if we define an API, then all the access of same context should use the APIs.